OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -621,9 +621,9 @@ protected virtual void OnBecomingIdle(Actor self)
 				self.QueueActivity(new TakeOff(self));
 			else if (Info.CanHover && self.Info.HasTraitInfo<AutoCarryallInfo>() && Info.IdleTurnSpeed > -1)
```

Comment:
Do we still need to check `AutoCarryallInfo` here? Isn't `Info.IdleTurnSpeed > -1` sufficient?

Reply:
It actually should be, yes.